README.md

```diff
@@ -79,46 +82,45 @@ echo "your keystore password" > `pwd`/password.txt
 
 Then run the faucet application without the wallet command-line flags:
 ```bash
-./lsk-faucet -httpport 8080
+make run FLAGS="-httpport 8080"
 ```
 
 **Optional Flags**
 
 The following are the available command-line flags(excluding above wallet flags):
 
-| Flag              | Description                                      | Default Value                              |
-| ----------------- | ------------------------------------------------ | ------------------------------------------ |
-| -httpport         | Listener port to serve HTTP connection           | 8080                                       |
-| -proxycount       | Count of reverse proxies in front of the server  | 0                                          |
-| -token-address    | Token contract address                           | 0x8a21CF9Ba08Ae709D64Cb25AfAA951183EC9FF6D |
-| -faucet.amount    | Number of LSK to transfer per user request       | 1                                          |
-| -faucet.minutes   | Number of minutes to wait between funding rounds | 10080 (1 week)                             |
-| -faucet.name      | Network name to display on the frontend          | sepolia                                    |
-| -faucet.symbol    | Token symbol to display on the frontend          | LSK                                        |
-| -hcaptcha.sitekey | hCaptcha sitekey                                 |                                            |
-| -hcaptcha.secret  | hCaptcha secret                                  |                                            |
+| Flag              | Description                                         | Default Value                              |
+| ----------------- | --------------------------------------------------- | ------------------------------------------ |
+| -httpport         | Listener port to serve HTTP connection              | 8080                                       |
+| -proxycount       | Count of reverse proxies in front of the server     | 0                                          |
+| -token-address    | Token contract address                              | 0x8a21CF9Ba08Ae709D64Cb25AfAA951183EC9FF6D |
+| -faucet.amount    | Number of ERC20 tokens to transfer per user request | 1                                          |
```

Comment:
Is it safe to assume that all ERC20 tokens can be divided into 10^18 parts (similar to 1 ETH == 10^18 Wei)?
If not, should we also add the support for `decimals`?

Reply:
Added support for this.